SalesforceCommerceCloud/pwa-kit
# V2 Loading Page

# Description

This is the loading screen that the users would see in their browser when they run `npm run start` in their app's directory. The script would initially start the dev server and make the build. When the build is done, the loading screen would disappear, as it redirects to their app's homepage.

For more details, see [ticket](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000szlPbYAI/view).

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# How to Test-Drive This PR

1. From the root directory, run `npm ci`.
2. Switch to the `pwa` package directory
3. Run `npm run start`
4. Verify that loading screen shows up in your browser
5. And when the 1st build is done, the loading screen would redirect to the homepage

# Checklists


## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
